packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md

```diff
@@ -172,3 +289,14 @@ To specify Binding Members, use the REST URL of the corresponding Okta API:
 | `okta.apps.manage`                      | Allows the admin to fully manage apps and their members in your Okta organization                                                                    | All Apps, All apps of specific type, a specific App |
 | `okta.apps.assignment.manage`           | Allows the admin to only manage assignment operations of an app in your Okta org                                                                     | All Apps, All apps of specific type, a specific App |
 | `okta.profilesource.import.run`         | Allows the admin to run imports for apps with a profile source, such as HRaaS and AD/LDAP apps. Admins with this permission can create users through the import. | All Apps, All apps of specific type, a specific App |
+
+#### Early Access Permission types
+
+<ApiLifecycle access="ea" />
+
+| Permission type                         | Description                                                                                                                                           | Applicable resource types                    |
+| :-------------------------------------- | :---------------------------------------------------------------------------------------------------------------------------------------------------- | :------------------------------------------- |
+| `okta.authzServers.read`                | Allows the admin to read authorization servers.                                                                                                      | All authorization servers, a specific authorization server |
+| `okta.authzServers.manage`              | Allows the admin to manage authorization servers.                                                                                                    | All authorization servers, a specific authorization server |
+| `okta.customizations.read`              | Allows the admin to read customizations.                                                                                                             | All customizations |
+| `okta.customizations.manage`            | Allows the admin to manage customizations.                                                                                                           | All customizations |
```

Comment:
```suggestion
| `okta.customizations.manage` <ApiLifecycle access="ea" />  | Allows the admin to manage customizations                                                                                                           | All customizations |
```
Sorry for the nit. We don't put a period for a one liner fragment, as per our style guide.